tests/test_compare.py

```diff
@@ -0,0 +1,30 @@
+import sys
+import os
+from pychimera import patch_environ, enable_chimera
+
+patch_environ()
+# enable_chimera()
+
+import chimera
+from compare_equal import compare
+
+
+def test_pdb():
+    """Test fot the function compare_pdb"""
```

Comment:
Careful with typos :)

Reply:
Corrected:
```suggestion
    """Test for the function compare_pdb"""
```

Reply:
Corrected: `"""Test for the function compare_pdb"""`